serial_no: "841612070098"
topic_name: realsense_lidar